ci/jenkins_build.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
+set -euo pipefail
+
+source ../utils.sh
+
+# This script does the following in sequence:
+# - Runs a Helm install of a Conjur OSS server
+# - Runs a Helm test that deploys a test container that runs a
+#   Bash Automated Test System (a.k.a. "Bats") test script that
+#   confirms that the Conjur server's status page is active.
+#
+# Optional Environment Variables:
+#   CONJUR_NAMESPACE:     Namespace to use for Conjur deployment. The
+#                         namespace is created if it doesn't exist.
+#   HELM_INSTALL_TIMEOUT: Helm install timeout in seconds.
+#                         Defaults to `180`.
+#   HELM_TEST_LOGGING:    Set to true to enable Helm test log collection.
+#                         Defaults to false.
+#   HELM_VERSION:         Helm client version to use for the test.
+#                         Defaults to '3.1.3'.
+#   KUBECTL_VERSION:      Kubectl client version to use for the test.
+#                         Defaults to '1.16.9'.
+#   SKIP_GCLOUD_LOGIN:    If set to 'true', then skip Gcloud authentication.
+#                         This is useful for local testing you've already
+#                         authenticated with GCP and/or have 'kubectl'
+#                         access to a cluster. Defaults to 'false'.
+
+test_id="$(random_string)"
+export CONJUR_NAMESPACE="${CONJUR_NAMESPACE:-conjur-oss-test-$test_id}"
+export HELM_INSTALL_TIMEOUT="${HELM_INSTALL_TIMEOUT:-180}"
+export HELM_TEST_LOGGING="${HELM_TEST_LOGGING:-true}"
+export HELM_VERSION="${HELM_VERSION:-3.1.3}"
+export KUBECTL_VERSION="${KUBECTL_VERSION:-1.16.9}"
+export SKIP_GCLOUD_LOGIN="${SKIP_GCLOUD_LOGIN:-false}"
+# Use release name not based on timestamp since tests can run in parallel
+export RELEASE_NAME="$CONJUR_NAMESPACE"
+
+announce "Building gcloud/kubectl/helm client image..."
+tmp_dir="$(pwd)/.tmp"
+bin_dir="${tmp_dir}/bin"
```

Comment:
For maintainability, this should indicate more what it is (maybe `tmp_bin_dir`?)